### ISSUE TITLE
Fig integration with WebStorm in my system is not working

### ISSUE THREAD

**Issue by rijans** [withfig/fig#2088]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 11.7.1 (20G918)

### Expected behaviour

Fig integration with WebStorm Terminal

### Actual behaviour

Fig integration with WebStorm in my system is not working

### Steps to reproduce

Open terminal in Webstorm, type anyting for suggestions.

### Environment

```yaml
fig-details:
  - 2.7.9
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz
  - cores: 2
  - mem: 8.00 GB
os-info:
  - macOS 11.7.1 (20G918)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/jaber
  - exe-path: /Users/jaber/.local/bin/fig
  - install-method: brew
  - env-vars:
    - FIGTERM_SESSION_ID: d09bff70-b8ea-4f95-8976-a74a42fe5f78
    - FIG_HOSTNAME: Jabers-MacBook-Pro.local
    - FIG_TERM: 2.7.9
    - PATH: /Users/jaber/.nvm/versions/node/v12.22.12/bin:/Users/jaber/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/jaber/.fig/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PID: 3907
    - FIG_PARENT: d09bff70-b8ea-4f95-8976-a74a42fe5f78
    - FIG_SET_PARENT: d09bff70-b8ea-4f95-8976-a74a42fe5f78
```

**Comment by grant0417** [comment]
We have a fix in progress for the WebStorm not working

**Comment by fabio-alkimia** [comment]
PhpStorm too..

**Comment by grant0417** [comment]
This should be resolved now in Fig 2.10, let me know if you continue to have any issues!
